via/services/jstor.py

```diff
@@ -0,0 +1,31 @@
+from via.requests_tools.headers import add_request_headers
+from via.services import HTTPService
+
+
+class JSTORAPI:
+    DEFAULT_DOI_PREFIX = "10.2307"
```

Comment:
This could do with a comment to indicate that is JSTOR's main DOI prefix, and that it is used both for real DOIs and pseudo-DOIs which have the format `10.2307/{jstor_article_id}` but are not actually registered DOIs.